science/gildas/Portfile

```diff
@@ -7,6 +7,7 @@ PortGroup           active_variants 1.1
 name                gildas
 version             202403a
 set my_version      [string tolower [clock format [clock scan 2000-[string range ${version} 4 5]-10] -format %b]][string range ${version} 2 3][string range ${version} 6 end]
+set my_env_vers     [string tolower [exec sw_vers -productName | tr -d " "]]
```

Comment:
This will execute `sw_vers` every time the port is parsed, slowing down portindexing and also failing to index the port on all non-Mac operating systems since they don't have the `sw_vers` program. This will cause the port to disappear from the ports.macports.org web site, for example, since its backend processes run portindex and do not run on macOS. Please arrange to run `sw_vers` only when someone is actually installing the port. And for maximum correctness, do that only on macOS.

Reply:
Thanks for your advises. I think I addressed all of them now, except the port is intended to work only on macOS (we have other solutions for other operating systems; if this changes in the future I will do the necessary).